dataland-frontend/src/components/pages/DatasetOverview.vue

```diff
@@ -1,8 +1,8 @@
 <template>
   <AuthenticationWrapper>
     <TheHeader />
-    <DatasetsTabMenu :initial-tab-index="1">
-      <TheContent class="p-3 min-h-screen paper-section relative">
+    <TheContent class="paper-section relative">
```

Comment:
Die Reihenfolge scheint mir jetzt sinnvoller als vorher. Kann mir trotzdem jemand kurz erklären wie ihr drauf gekommen seid?

Reply:
Das weiß ich leider nicht. Kannst du mal über "blame" gucken, ob das vllt Marcin war? Dann könntest du ihn fragen.

Reply:
Eben mit Marcin gesprochen, die Änderung kommt daher damit der Footer immer am unteren Teil der Seite ist.
Jetzt frage ihc mich ob es sinnvoll wäre, das auch noch an der letzten Stelle in "SearchCompaniesForFrameworkData" auch anzupassen? Was meinst du?

Reply:
Ich schau mir mal die STelle an und sag dir Bescheid

Reply:
Da dies keine Kernfunktionalität für den PR betrifft schließe ich den Kommentar. Die vorgeschlagene Änderungen wird dann in einem zukünftigen feature branch evaluiert/implementiert.